src/commands/deployImage/IDeployImageContext.ts

```diff
@@ -6,9 +6,11 @@
 import type { ContainerApp, EnvironmentVar } from "@azure/arm-appcontainers";
 import type { ContainerRegistryManagementModels } from '@azure/arm-containerregistry';
 import { ISubscriptionActionContext } from '@microsoft/vscode-azext-utils';
-import { SupportedRegistries } from '../../constants';
+import { ImageSourceValues, SupportedRegistries } from '../../constants';
 
 export interface IDeployImageContext extends ISubscriptionActionContext {
+    deployMethod?: ImageSourceValues;
```

Comment:
Wondering if we can just use `imageSource` instead of `deployMethod` since `ImageSourceListStep` already handles setting this value for us.

Reply:
I think this is something I forgot to remove from when I had a fully separate deploy step from `ImageSourceListStep`.